August  news (#1348)
ropensci/roweb3: content/blog/2026-08-28-news-august-2026/index.Rmd

:dna: R-Universe is now part of the infrastructure of the [Bioconductor submission process](https://blog.bioconductor.org/posts/2026-06-15-new-submission-process-with-Runiverse/).

## We're still celebrating our 15th anniversary! 🎉

In July, we started to share stories from members of our community about their experiences with rOpenSci. Our first story features [Eunseop Kim](/author/eunseop-kim/) and his connection with rOpenSci.
Read it on our blog: [From Peer Review to Mentorship: My rOpenSci Story](/blog/2026/07/14/15yo-eunseop-kim/)
Stay tuned for more stories from our community as we continue celebrating 15 years of rOpenSci!

### Coworking 

Read [all about coworking](/blog/2023/06/21/coworking/)!

- Tuesday September 1st, 14:00 Europe Central (12:00 UTC) ["Getting to Know SORTEE"](/events/coworking-2026-09/), with [Steffi LaZerte](/author/steffi-lazerte) and co-host [Ed Ivimey-Cook](/author/ed-ivimey-cook/).
  - Visit [SORTEE](https://sortee.org/) (Society for Open, Reliable, and Transparent Ecology and Evolutionary Biology).
  - Meet co-host, Ed Ivimey-Cook, and learn more about SORTEE and how you might get involved.

>  

Reviewer: ```suggestion
 
- Tuesday October 6th, 09:00 Americas Pacific (16:00 UTC) ["Writing Tests & Testing in R"](/events/), with [Yanina Bellini Saibene](/author/yanina-bellini-saibene) and co-host Olivier Leroy.
  - Explore how to write tests for R and add some tests to your work or packages
  - Meet co-host, Olivier Leroy, and chat about testing

- Tuesday November 3rd, 09:00 Australia Western (01:00 UTC) [TBA](/events/), with [Steffi LaZerte](/author/steffi-lazerte) and co-host TBA.

 - Tuesday December 8th, 14:00 Europe Central (12:00 UTC) ["Code Linting in R"](/events/), with  [Steffi LaZerte](/author/steffi-lazerte) and co-host [Etienne Bacher](/author/etienne-bacher/).
  - Read up on Code Linting and apply some linters to your R code
  - Meet co-host, Etienne Bacher, and discuss code linting in general, or flir and Jarl in particular
\* Note that December coworking is a week later than usual

```

---
slug: "news-august-2026"
title: rOpenSci News Digest, August 2026
author:
  - The rOpenSci Team
date: '2026-08-28'
tags:
  - newsletter

> description: champions program; R-universe update; Quinceañera; new packages and package news

Reviewer: ```suggestion
description: Champions Program; R-Universe update; Quinceañera; new packages and package news
```

  last_newsletter: '2026-06-30'
  doi: "10.59350/4chwm-mtm35"
---

```{r setup, include=FALSE}
library("rlang")
last_newsletter <- anytime::anytime(params$last_newsletter)
knitr::opts_chunk$set(echo = FALSE)
url <- sprintf(
  "/blog/%s/%s/%s/%s",
  lubridate::year(rmarkdown::yaml_front_matter(knitr::current_input())$date),
  stringr::str_pad(
    lubridate::month(rmarkdown::yaml_front_matter(knitr::current_input())$date),
    2,
    "0",
    side = "left"
  ),
  stringr::str_pad(
    lubridate::day(rmarkdown::yaml_front_matter(knitr::current_input())$date),
    2,
    "0",
    side = "left"
  ),
  rmarkdown::yaml_front_matter(knitr::current_input())$slug
)
english <- function(x) {
  as.character(english::english(x))
}

nice_string <- function(...) {
  if (length(...) == 2) {
    return(paste(c(...), collapse = " and "))
  }
  glue::glue_collapse(trimws(...), sep = ", ", last = ", and ")
}
```
<!-- Before sending DELETE THE INDEX_CACHE and re-knit! -->

Dear rOpenSci friends, it's time for our monthly news roundup!
<!-- blabla -->
You can read this post [on our blog](`r url`).
Now let's dive into the activity at and around rOpenSci!

## rOpenSci HQ

### Champions Program update

> Our Champions are making great progress! 🌟 
> They’ve now completed the training on good open source software development practices, package development, and peer review, and are moving on to explore community building and communications.
> Meanwhile, mentoring is underway, with monthly meetings helping Champions move their projects forward. We’re already starting to see some exciting results: first versions of packages are taking shape, and some Champions will soon be sharing their work at LatinR!
> Stay tuned for more updates as their projects continue to grow! 🚀

Reviewer: ```suggestion
Our Champions are making great progress! 🌟 
They've now completed the training on good open source software development practices, package development, and peer review, and are moving on to explore community building and communications.
Meanwhile, mentoring is underway with monthly meetings helping Champions move their projects forward. We're already starting to see some exciting results: first versions of packages are taking shape, and some Champions will soon be sharing their work at LatinR!
Stay tuned for more updates as their projects continue to grow! 🚀
```

### R-Universe updates

> :muscle: R-universe has started building and checking packages for Windows ARM64. 

Reviewer: ```suggestion
:muscle: R-Universe has started building and checking packages for Windows ARM64.
```

Read more in our [tech note](/blog/2026/08/06/r-universe-winarm/).

> :dna: R-universe is now part of the infrastructure of the [Bioconductor submission process](https://blog.bioconductor.org/posts/2026-06-15-new-submission-process-with-Runiverse/).

Reviewer: ```suggestion
:dna: R-Universe is now part of the infrastructure of the [Bioconductor submission process](https://blog.bioconductor.org/posts/2026-06-15-new-submission-process-with-Runiverse/).
```

## We're still celebrating our 15th anniversary! 🎉

In July, we started to share stories from members of our community about their experiences with rOpenSci. Our first story features [Eunseop Kim](/author/eunseop-kim/) and his connection with rOpenSci.
Read it on our blog: [From Peer Review to Mentorship: My rOpenSci Story](/blog/2026/07/14/15yo-eunseop-kim/)
Stay tuned for more stories from our community as we continue celebrating 15 years of rOpenSci!

### Coworking 

Read [all about coworking](/blog/2023/06/21/coworking/)!

> - Tuesday September 1st, 14:00 Europe Central (12:00 UTC) ["Getting to Know SORTEE"](/events/coworking-2026-09/), with [Steffi LaZerte](/author/steffi-lazerte) and cohost [Ed Ivimey-Cook](/author/ed-ivimey-cook/).

Reviewer: I'll have to try harder to be more consistent with the hyphen!
```suggestion
- Tuesday September 1st, 14:00 Europe Central (12:00 UTC) ["Getting to Know SORTEE"](/events/coworking-2026-09/), with [Steffi LaZerte](/author/steffi-lazerte) and co-host [Ed Ivimey-Cook](/author/ed-ivimey-cook/).
```

  - Visit [SORTEE](https://sortee.org/) (Society for Open, Reliable, and Transparent Ecology and Evolutionary Biology).

>   - Meet co-host, Ed Ivimey-Cook, and learn more about SORTEE and how you might get involved.

Reviewer: Hoping to have more coworking here, but I'm not sure I'll have it in time 😓

packages <- split(registry, seq(nrow(registry)))
get_release <- function(repo) {
  info <- gh::gh(
    "GET /repos/{owner}/{repo}/releases",
    owner = repo$owner,
    repo = repo$repo,
    per_page = 3,
    description = repo$description
  )
  info <- info[!purrr::map_lgl(info, "draft")]
  if (
    length(info) == 0 ||
      anytime::anytime(info[[1]]$published_at) < last_newsletter
  ) {
    return(NULL)
  }

  tibble::tibble(
    package = repo$package,
    version = info[[1]]$tag_name,
    url = info[[1]]$html_url,
    description = repo$description
  )
}
releases <- purrr::map_df(
  packages,
  get_release
)
releases <- split(releases, seq(nrow(releases)))
format_release <- function(release) {
  sprintf(
    '[%s](https://docs.ropensci.org/%s "%s") ([`%s`](%s))',
    release$package,
    release$package,
    release$description,
    release$version,
    release$url
  )
}
all_releases <- purrr::map_chr(releases, format_release)
text <- nice_string(all_releases)
```

The following `r if (length(releases) > 1) english(length(releases))` package`r if (length(releases) > 1) "s"` `r if (length(releases) > 1) "have" else "has"` had an update since the last newsletter: `r text`.

> The writexl package has a [new maintainer](https://github.com/ropensci/writexl/pull/98#issuecomment-4191858158), Bill Denney.

Reviewer: I think it would be nice to celebrate these maintainers a bit more; could we highlight this a bit more emphatically and celebrate these handovers? Perhaps give it a separate heading and say something like "We welcome new maintainers to two new packages... Thank you for taking on the tasks!" or something similar? We could also thank the past maintainers for their contributions as well.

I'd also consider moving this to be close to the "Calls for contributions" section.

Reviewer 2: I like this suggestion.

Author: I need to remove this though, this is from last month :see_no_evil:

      collapse = "\n\n"
    )
  )
  cat("\n\n")
}

others <- posts[purrr::map_lgl(posts, ~ any(.x[["other"]]))]
if (length(others) > 0) {
  cat(
    paste0(
      purrr::map_chr(others, format_post),
      collapse = "\n\n"
    )
  )
  cat("\n\n")
}

tech_notes <- posts[purrr::map_lgl(posts, ~ any(.x[["tech_note"]]))]
if (length(tech_notes) > 0) {
  cat("\n\n")
  cat("### Tech Notes\n\n")
  cat(
    paste0(
      purrr::map_chr(tech_notes, format_post),
      collapse = "\n\n"
    )
  )
  cat("\n\n")
}
```

## Calls for contributions

### Calls for maintainers

If you're interested in maintaining any of the R packages below, you might enjoy reading our blog post [What Does It Mean to Maintain a Package?](/blog/2023/02/07/what-does-it-mean-to-maintain-a-package/).

- [charlatan](https://docs.ropensci.org/charlatan), create fake data in R.
  [Issue for volunteering](https://github.com/ropensci/charlatan/issues/150).

### Calls for contributions

Refer to our [help wanted page](/help-wanted/) -- before opening a PR, we recommend asking in the issue whether help is still needed.

## Package development corner

Some useful information for R package developers.
:eyes:

> ## From Rd files to Quarto

Reviewer: ```suggestion
### From Rd files to Quarto
```

Edgar Ruiz from Posit released [pkgsite](https://opensource.posit.co/blog/2026-06-18_pkgsite-0-1-0/),
a package for converting your package's `.Rd` files to Quarto.
It creates qmd files that you can integrate as you want in a Quarto website.

> ## Mutation testing, fuzzy testing

Reviewer: ```suggestion
### Mutation testing, fuzzy testing
```

First of all, a reminder in case you confuse the two concepts...

👽️ Mutation testing: you run tests on mutated version of the *code*. 

🚮 Fuzzy testing: you run tests with all sorts of *inputs* to the code.

At useR! 2026, both topics were covered:

- mutation testing in [Beyond Code Coverage: Mutation Testing in R with mutator](https://docs.google.com/presentation/d/1T5iK0gRBFF869Q6eirtmYVV-5ABD4EN-WBJI5gl5EUY/edit?slide=id.g3efebd0f55a_0_99#slide=id.g3efebd0f55a_0_99) by Assanali Amandykov, Pierre Donat-Bouillud.
- fuzzy testing in [Fuzz-testing R-based research software for robustness](https://events.digital-research.academy/event/109/contributions/430/attachments/187/407/slides-v2.pdf) by Marco Colombo.

> ## roxyreqs

Reviewer: ```suggestion
### roxyreqs
```

Also at useR!, Moritz Lang and collaborators introduced [roxyreqs](https://github.com/mnlang/roxyreqs), a package for adding roxygen2-like documentation to testthat.
[Slides](https://events.digital-research.academy/event/109/contributions/471/attachments/69/201/roxyreqs-user2026.pdf).

> ## checktor, a new helper for CRAN submissions

Reviewer: ```suggestion
### checktor, a new helper for CRAN submissions
```

If you want to submit your package to CRAN, you can get help through the [CRAN cookbook](https://contributor.r-project.org/cran-cookbook/), the [collaborative list maintained by ThinkR](https://github.com/ThinkR-open/prepare-for-cran) and now a new package, checktor by James Balamuta!
Read more in [James' post](https://blog.thecoatlessprofessor.com/programming/r/the-check-passed-the-reviewer-didnt/).

> ## Interesting AI reads

Reviewer: ```suggestion
### Interesting AI reads
```
